src/Tests/CoreTests/Geometry/GeometryTests.hpp

```diff
@@ -36,7 +36,7 @@ namespace RaTests
 
             // Test line queries.
             RA_UNIT_TEST(  pointToLineSq(a, a, b-a) == 0.f, "distance from A to AB" );
-            RA_UNIT_TEST(  pointToLineSq(b, a, b-a) == 0.f, "distance from B to AB" );
+            RA_UNIT_TEST(  areApproxEqual(pointToLineSq(b, a, b-a), 0.f), "distance from B to AB" );
```

Comment:
Why changing this test only and not the first one too?

Reply:
right, I will fix that (i'm only responsible for the rebase/subdivider app, I do not rewrite nor read all the code ;))

Reply:
btw it should be equal or approx equal ?
if approxEqual I should change the segment tests also